### PR TITLE
List available streams

### PR DESCRIPTION
**Pull request by adrn**
This might be a very simple way of addressing #5. But: how do I test this locally?!

### CODE REVIEW

**Comment by dfm** [comment]
Do the steps listed in the readme not work for you? I realize that they're a bit telegraphic :)

2 suggestions:

1. We might want `playing_to` instead of `listening_to` because we'll want to show rooms even if there are no listeners.

2. Take a look at [the admin template](https://github.com/dfm/spotify-party/blob/master/src/spotify_party/templates/admin.html) for an example of how to format the urls properly.

Finally, I wouldn't mind have an option for private streams but we can add that later.

**Comment by adrn** [comment]
How do you want to format the room names in the displayed list? Right now it looks like this:
image

But it seems to be working now!

**Comment by adrn** [comment]
Also, if we want to add support for private rooms, I feel like we should add a `Room` table to the database...

File: src/spotify_party/views.py [hunk 1]
```diff
@@ -109,6 +109,20 @@
     )
 
 
+@routes.get("/rooms", name="rooms")
+@api.require_auth
+async def rooms(request: web.Request, user: db.User) -> web.Response:
+    rooms = await request.app["db"].get_all_rooms()
+    if rooms is None:
+        return aiohttp_jinja2.render_template("notfound.html", request, {})
```
Comment (dfm) [inline src/spotify_party/views.py:117]: This should probably be:

```python
return web.HTTPNotFound()
```

for the same effect!

**Comment by dfm** [comment]
Awesome! I'll merge this as is and we can consider a separate table. My only hesitation is that it might be a bit annoying to keep synchronized and cleaned up (we want a user to only ever play to one room at a time) but it definitely could come in handy.

**Comment by dfm** [comment]
Also: I think we should just put this at `/listen` instead of `rooms`. What do you think?

**Comment by adrn** [comment]
All sounds good to me!

**Pull request merged**